centml/cli/cluster.py

```diff
@@ -71,43 +108,146 @@ def get(id):
     )
 
     click.echo("Additional deployment configurations:")
-    click.echo(
-        tabulate(
-            [
-                ("Is private?", deployment.secrets is not None),
-                ("Hardware", id_to_hw_map[deployment.hardware_instance_id]),
-                ("Port", deployment.port),
-                ("Healthcheck", deployment.healthcheck or "/"),
-                ("Replicas", {"min": deployment.min_replicas, "max": deployment.max_replicas}),
-                ("Environment variables", deployment.env_vars or "None"),
-            ],
-            tablefmt="rounded_outline",
-            disable_numparse=True,
+    if type == platform_api_client.DeploymentType.INFERENCE:
+        click.echo(
+            tabulate(
+                [
+                    ("Port", deployment.port),
+                    ("Healthcheck", deployment.healthcheck or "/"),
+                    ("Replicas", {"min": deployment.min_replicas, "max": deployment.max_replicas}),
+                    ("Environment variables", deployment.env_vars or "None"),
+                    ("Max concurrency", deployment.timeout or "None"),
```

Comment:
we need to rename this field to concurrency or something instead of timeout.

Can you create a ticket for this one as well?

Reply:
yes i can create a ticket but this one would require changes in platform backend codebase too in order to rename the schema field from timeout to concurrency

Reply:
https://github.com/CentML/centml-python-client/issues/51